### DISCUSSION TITLE
VTX administrator to follow model match

### DISCUSSION THREAD

**Discussion by chrysburge** [ExpressLRS/ExpressLRS#2404] [Q&A] [answered]
Got a question. When will VTX administrator follow model match? 
This would be very handy for me as a quad pilot. I have many quads using ELRS that have different VTX hardware. Every time I switch models, I have to adjust the VTX parameters in the VTX Administrator for that specific model (mainly the power level).
Is this something that can be implemented into the code or is it a hardware limitation or something else? 
Thank you for any guidance.

**Comment by pkendall64** [comment, marked as answer]
It was a decision by the developer to have the VTX settings be global, rather than per model. The reasoning is that at a race event you would not want to switch model and have the VTX jump over to the configured channel for that model and stomp on someone else that is flying causing them to crash.

**Reply by chrysburge** [reply]
Makes sense. 
I wonder if there could be an option to toggle on the functionality that I am inquiring about.?
Thank you for your reply and consideration.
